### PR TITLE
MERC-1175 the fix for the dropdown bug using blueprint method

### PR DESCRIPTION
**Pull request by mcampa**
I found the solution in blueprint javascript

Is a jquery plugin that handles the hiding by creating a shadow hidden select element.

@bc-ranji @bc-ong @bc-julianleong

### CODE REVIEW

File: assets/js/theme/common/select-option-plugin.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * Visually hides the option from user by moving option to an invisible
+ * and disabled select placeholder element.
+ *
+ * This approach is required rather than simply hiding the option because
+ * hidden option can still be included when serializeArray() is called and
+ * cause wrong value to be submitted.
+ * (eg. if you have option 1, 2, 3 and 2 is hidden, when you select 3,
+ * serializeArray() will use the value of 2 instead of 3)
+ */
+$.fn.toggleOption = function (show) {
```
Comment (mcampa) [inline assets/js/theme/common/select-option-plugin.js]: This is a copy of blueprint code that handles the hiding/showing of options in a dropdown

**Comment by bc-ranji** [comment]
💚

**Comment by mcampa** [comment]
@bc-ong

**Comment by bc-ong** [comment]
:+1: LGTM

**Pull request merged**